src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -217,7 +217,10 @@ function getDrupalContent(buildOptions) {
       buildOptions.drupalError = drupalData;
       log(err.stack);
       log('Failed to pipe Drupal content into Metalsmith!');
-      if (buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST) {
+      if (
+        buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST ||
+        buildOptions['drupal-fail-fast']
```

Comment:
@jonpugh It looked like this was the easiest way to catch all the scenarios where fetching the content from Drupal failed in some way. Previously, we'd throw an error for any environment except localhost. With this flag now, even localhost will throw the error, breaking the build.

Reply:
Sweet!

Does this check also handle the need to disable the proxy? I think in our CI, it was erroring there because the app thought it needed the Proxy when it did not.